src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.ts

```diff
@@ -43,6 +48,20 @@ export class UbsClientProfilePageComponent implements OnInit {
     });
   }
 
+  public openDeleteProfileDialog() {
+    const dialogRef = this.dialog.open(UbsProfileDeletePopUpComponent, {
+      hasBackdrop: true
+    });
+    dialogRef.afterClosed();
+  }
+
+  public openChangePasswordDialog() {
+    const dialogRef = this.dialog.open(UbsProfileChangePasswordPopUpComponent, {
+      hasBackdrop: true
+    });
+    dialogRef.afterClosed();
```

Comment:
afterClosed returns observables, so do we need this here ?

Reply:
fixed